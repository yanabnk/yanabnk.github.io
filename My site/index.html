<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <!-- Шрифты -->
    <link href="https://fonts.googleapis.com/css?family=Cardo:400i|Open+Sans:700|Raleway:300,400,600,700&display=swap&subset=cyrillic-ext" rel="stylesheet">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Bootstrap -->
    <link rel="stylesheet" href="bootstrap/css/bootstrap.min.css">

    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"> <!--для адаптивности -->
   <!-- Favicon  https://realfavicongenerator.net/-->
    <link rel="apple-touch-icon" sizes="180x180" href="img/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="img/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="img/favicon/favicon-16x16.png">
    <link rel="manifest" href="img/favicon/site.webmanifest">
    <link rel="mask-icon" href="img/favicon/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="msapplication-TileColor" content="#da532c">
    <meta name="theme-color" content="#ffffff">
<!-- Иконки -->
    <script src="https://kit.fontawesome.com/8c5f8983b2.js" crossorigin="anonymous"></script>
   <!--  -->
    <meta name="description" content="Описание-страницы-кратко"> <!-- для гугл -->
    <meta name="keywords" content="Описание-страницы-кратко"> <!-- для яндекс -->

    <meta name="google" content="notranslate" />

    <title>BNK</title>

    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
</head>
<body>

<!--header-->

    <header class="header" id="header">
        <div class="container">
            <div class="header-inner">
                <div class="header-logo">
                    <a class="nav-link logo" href="#">BNK</a>
                </div>
                <nav class="nav" id="nav">
                    <a class="nav-link" href="#" data-scroll="#features">Обо Мне</a>
                    <a class="nav-link" href="#" data-scroll="#works">Фильмы</a>
                    <a class="nav-link" href="#" data-scroll="#emotion">Эмоции</a>
                    <a class="nav-link" href="#" data-scroll="#reviews">Родина</a>
                    <a class="nav-link" href="#" data-scroll="#download">Фраза</a>
                    <a class="nav-link" href="#" data-scroll="#table">Предметы</a>
                </nav>

                <button class="burger" type="button" id="navToggle">
                    <span class="burger-item">Menu </span>
                </button>

            </div>
        </div>
    </header>

<!--Intro-->

 <div class="intro" id="intro">
     <div class="container">
         <div class="intro-inner">
         <h1 class="intro-title">Yana Budnichenko</h1>
         <h2 class="intro-subtitle">+380-63-802-80-04</h2>
         <h2 class="intro-subtitle">Харьков, Украина</h2>
         <a href="mailto:yanabnk@gmail.com" class="btn btn-red">Write me</a>
        </div>
 </div>  
 </div> 


 <!-- Features -->
 <div class="character container" id="features">Характеристика в Трёх Словах</div>
<div class="container" >
    <div class="features">
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-1.png" alt="">
            <h4 class="features-title">Инициативная</h4>
            <div class="features-text">Не жди, когда тебя выберут или спросят
            </div>
        </div>
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-4.png" alt="">
            <h4 class="features-title">Коммуникабельная</h4>
            <div class="features-text">Умение вести диалог — это талант.
            </div>
        </div>
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-5.png" alt="">
            <h4 class="features-title">Ответственная</h4>
            <div class="features-text">Ответственность — это та цена, которую мы платим за власть.
            </div>
        </div>
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-2.png" alt="">
            <h4 class="features-title">Доброжелательная</h4>
            <div class="features-text">
                Избыток доброжелательности ведет к тому, что в один прекрасный день вы оказываетесь
                в очень плохом окружении; избыток проницательности ведет к полнейшему одиночеству.
            </div>
        </div>
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-6.png" alt="">
            <h4 class="features-title">Пунктуальная</h4>
            <div class="features-text">Пунктуальность — умение угадывать, на сколько опоздает другой.
            </div>
        </div>
        <div class="features-item">
            <img class="features-icon" src="img/features/feature-3.png" alt="">
            <h4 class="features-title">Дружелюбная</h4>
            <div class="features-text">Дружелюбие – это способность терпеть дураков с радостью.
            </div>
        </div>
        
    </div> <!--./features-->
</div><!--./container-->


<!--FILMS-->

<div class="film" id="works">100+ Любимых фильмов</div>
<div class="films" >
    <div class="films-item">
        <img class="films-foto" src="img/films/film-1.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Вечное сияние чистого разума</div>
            <div class="films-text">«Можно стереть любовь из памяти. Выкинуть из сердца — это уже другая история»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-2.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Иллюзия обмана</div>
            <div class="films-text">«Чем вы ближе, тем меньше вы видете»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-3.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Гарри Поттер</div>
            <div class="films-text">«Путешествие в твою мечту»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-4.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Хоббит</div>
            <div class="films-text">«From the smallest beginnings come the greatest legends.»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-5.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Список Шиндлера</div>
            <div class="films-text">«Этот список - жизнь»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-6.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Отпуск по Обмену</div>
            <div class="films-text">Описание чего-то</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-7.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Сплит</div>
            <div class="films-text">«Внутри Кевина живут 23 личности. Вскоре еще одна явит себя миру»</div> 
        </div>
    </div>

    <div class="films-item">
        <img class="films-foto" src="img/films/film-8.jpg" alt="">
        <div class="films-content">
            <div class="films-title">Век Адалин</div>
            <div class="films-text">«За последний век мир изменился. Адалин - нет»</div> 
        </div>
    </div>
</div><!--/films-->


<!-- EMOTION -->

<div class="emotion" id="emotion">

    <div class="container">
        <div class="emotion-title">Эмоции</div>
        <div class="emotion-inner">

            <div class="emotion-item">
                <img class="emotion-foto" src="img/emotion/emotion-1.jpg" alt="">
                <div class="emotion-name">Счастье</div>
                <div class="emotion-prof">Виктор Гюго</div>
                <div class="emotion-text"><p>Величайшее в жизни счастье — это уверенность в том, что нас любят, 
                    любят за то, что мы такие, какие мы есть, или несмотря на то, 
                    что мы такие, какие мы есть.</p></div>
            </div>

            <div class="emotion-item">
                <img class="emotion-foto" src="img/emotion/emotion-2.jpg" alt="">
                <div class="emotion-name">Грусть</div>
                <div class="emotion-prof">Винсент Ван Гог</div>
                <div class="emotion-text"><p>Грусть и разочарование ещё больше, чем распущенность, вредят нам, 
                    счастливым обладателям надорванных сердец.</p></div>
                <div class="emotion-social"></div>
            </div>

            <div class="emotion-item">
                <img class="emotion-foto" src="img/emotion/emotion-3.jpg" alt="">
                <div class="emotion-name">Занятость</div>
                <div class="emotion-prof">Генрих Гейне</div>
                <div class="emotion-text"><p>Не занятый делом человек никогда не может наслаждаться полным счастьем. 
                    На лице бездельника вы всегда найдёте отпечаток недовольства и апатии.</p></div>
                <div class="emotion-social"></div>
            </div>
        </div>
    </div>
</div>



<!-- Testimonials -->
<div class="reviews" id="reviews">
    <div class="container">

    <div class="reviews-slider" id="reviewsSlider">
     <div class="slick-item">
        <div class="reviews-item">
            <div class="reviews-photo">
                <img class="reviews-img" src="img/sliders/slide-1.jpg" alt="">
            </div>
            <div class="reviews-content">
                <div class="reviews-text">Харьков – не тот город, у которого душа нараспашку. Чтобы понять его, 
                    нужно присмотреться к нему повнимательнее.</div>
                <div class="reviews-author">Kharkiv, Ukraine</div>     
            </div>
        </div>
    </div>

    <div>
        <div class="reviews-item">
            <div class="reviews-photo">
                <img class="reviews-img" src="img/sliders/slide-2.jpg" alt="">
            </div>
            <div class="reviews-content">
                <div class="reviews-text">Родина мать - монументальная скульптура в Киеве на правом берегу Днепра.</div>
                <div class="reviews-author">Kyiv, Ukraine</div>     
            </div>
        </div>
    </div>

    <div>
        <div class="reviews-item">
            <div class="reviews-photo">
                <img class="reviews-img" src="img/sliders/slide-3.jpg" alt="">
            </div>
            <div class="reviews-content">
                <div class="reviews-text">Львов, является неоспоримым столицей украинской культуры, духовности и национальной идентичности</div>
                <div class="reviews-author">Lviv, Ukraine</div>     
            </div>
        </div>
    </div>

    <div>
        <div class="reviews-item">
            <div class="reviews-photo">
                <img class="reviews-img" src="img/sliders/slide-4.jpg" alt="">
            </div>
            <div class="reviews-content">
                <div class="reviews-text">Где соединяется несоединимое, где украинская степь встречается
                    с Черным морем, где чистый и свежий воздух, где живут веселые и остроумные люди.</div>
                <div class="reviews-author">Odessa, Ukraine</div>     
            </div>
        </div>
    </div>

    <div>
        <div class="reviews-item">
            <div class="reviews-photo">
                <img class="reviews-img" src="img/sliders/slide-5.jpg" alt="">
            </div>
            <div class="reviews-content">
                <div class="reviews-text">Украинское село славится особым колоритом и красотой. Здесь наша культура, 
                    обряды и обычаи сохранились и ценятся, как нигде.</div>
                <div class="reviews-author">Rivne, Ukraine</div>     
            </div>
        </div>
    </div>
    
    </div>

    </div>
</div> <!-- /.reviews -->


<!-- PHRASE -->
<div class="phrase" id="download">
    <div class="container">
        <h3 class="phrase-title ">Важная Фраза</h3>
        <div class="phrase-text">
            <p>Не тратьте деньги на одежду, тратьте деньги на путешествия.
                Какая разница, сколько лет Вашим кедам, если Вы гуляете в них по Парижу.</p>
        </div>
        <a class="btn btn-long btn-red" href="http://atkritka.com/279827/">Неизвестный автор</a>
    </div>
</div>


 <!-- UNIVERSITY -->

 <div class="table" id="table">
    <div class="container">
   <h2><a id="fourth">Cписок Изученых Предметов</a></h2>
<table class="table">
   <thead class="title">
     <tr>
       <th scope="col">#</th>
       <th scope="col">Предмет</th>
       <th scope="col">Преподаватель</th>
       <th scope="col">Оценка</th>
     </tr>
   </thead>
   <tbody>
     <tr class="line-color">
       <th scope="row">1</th>
       <td>Основы графической композиции</td>
       <td>Глибко</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">2</th>
       <td>АВС</td>
       <td>Ломакин</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">3</th>
       <td>Начертательная геометри</td>
       <td>Федченко</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">4</th>
       <td>Дискретная математика</td>
       <td>Татаринова</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">5</th>
       <td>Иностранный язык</td>
       <td>Ковтун</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">6</th>
       <td>Правоведение</td>
       <td>Семко</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">7</th>
       <td>Физическое воспитание</td>
       <td>Донец</td>
       <td>А</td>
     </tr>
     <tr class="line-color">
       <th scope="row">8</th>
       <td>ООП</td>
       <td>Водка</td>
       <td>А</td>
     </tr>
   </tbody>
 </table>
</div>
</div>

<!-- FOOTER -->
<div class="footer">
    <div class="container">

        <div class="footer-inner">
            <div class="footer-block">
                <h4 class="footer-title">BNK</h4>
                <address class="footer-address">
                   <div> <a href="index.html">Light</a></div>
                   <div> <a href="index-dark.html">Dark</a></div>
<!-- SWITCH -->
                   <style id="inverter" media="none">
                    html { background-color: #eee; filter: invert(100%) } 
                    * { background-color: inherit } 
                   img:not([src*=".svg"]), [style*="url("] { filter: invert(100%) }
                 </style>
               <label class="button">
                   <input type="checkbox" id="themer">
                   Dark <span aria-hidden="true"></span>
                 </label>

                </address>
            </div>


            <div class="footer-block">
                <h4 class="footer-title">BNK</h4>
                <address class="footer-address">
                   <div> <a href="https://www.instagram.com/yana_budnichenko/?hl=ru"> <i class="fab fa-instagram"></i>yana_budnichenko</a></div>
                   <div> <a href="https://www.facebook.com/miss.budnichenko"><i class="fab fa-facebook"></i>Яна Будниченко</a></div>
                </address>
            </div>

            <div class="footer-block">
                <h4 class="footer-title">BNK</h4>
                <div class="footer-text">
                   <div> <a href="index-3d.html">3D model</a></div>
                   <div> <a href="index.html">Home</a></div>
                </div>
            </div>

        </div>
    </div>

    <div class="footer-copyright">
        <div class="container">
        <div class="copyright-text">
           <div class=""> Copyright © 2020 BNK. <br>All Rights Reserved</div>
           <div class="">  Made <span> by Yana Budnichenko</span></div>
        </div>
    </div>

    </div>

</div>

<!-- Java -->
<script src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>

<script src="main.js"></script>
<script src="bootstrap/js/bootstrap.min.js"></script>

</body>
</html>