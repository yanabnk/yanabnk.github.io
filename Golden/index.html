<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="style.css">
    <title>Golden</title>
</head>
<body>

    <!-- HEADER -->
<div class="intro">
    <div class="container">
    <div class="header">
        <div class="header-logo">
        <a href="">Golden</a>    
        </div>
        <nav class="menu">
            <a href="">Home</a>
            <a href="">Services</a>
            <a href="">Portfolio</a>
            <a href="">About</a>
            <a href="">Contact</a>
        </nav>
    </div>
</div>


<!-- HOME -->

<div class="container">
    <div class="home-inner">
        <h3>Welcome To Our Studio!</h3>
        <h1>Itâ€™s nice to Meet you</h1>
        <a href="#" class="btn">Tell Me More</a>
    </div>
</div>
</div>


<!-- SERVICES -->
<div class="container">
    <div class="services">
    <div class="serv-title">
        <h2>Services</h2>
        <h4>Proin iaculis purus consequat sem cure. </h4>
    </div>
    <div class="serv">
        <div class="serv-item">
            <img src="img/Services/icon-1.jpg" alt="">
            <h2>E-Commerce</h2>
            <div class="serv-text">Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.</div>
        </div>
        <div class="serv-item">
            <img src="img/Services/icon-2.jpg" alt="">
            <h2>Responsive Web</h2>
            <div class="serv-text">Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.</div>
        </div>
        <div class="serv-item">
            <img src="img/Services/icon-3.jpg" alt="">
            <h2>Responsive Web</h2>
            <div class="serv-text">Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.</div>
        </div>
    </div>
</div>
</div>

<!-- PORTFOLIO -->
<div class="portfolio">
    <div class="container">
        <div class="ptf-title">
            <h1>Our portfolio</h1>
            <h2>Proin iaculis purus consequat sem cure</h2>
        </div>
        <div class="ptf-items">
        <div class="ptf-item">
            <img src="img/Portfolio/work-1.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Ebony & Ivory</h2>
                <h3>Branding</h3>
            </div>
        </div>
        <div class="ptf-item">
            <img src="img/Portfolio/work-2.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Smart Stationary</h2>
                <h3>Print Design</h3>
            </div>
        </div>
        <div class="ptf-item">
            <img src="img/Portfolio/work-3.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Clever Poster</h2>
                <h3>Print Design</h3>
            </div>
        </div>
        <div class="ptf-item ptf-item-four">
            <img src="img/Portfolio/work-4.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Vinyl Record</h2>
                <h3>Product Mock-Up</h3>
            </div>
        </div>
        <div class="ptf-item">
            <img src="img/Portfolio/work-5.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Treehouse Template</h2>
                <h3>Web Design</h3>
            </div>
        </div>
        <div class="ptf-item">
            <img src="img/Portfolio/work-6.jpg" alt="">
            <div class="ptf-item-text">
                <h2>Burned Logo</h2>
                <h3>Branding</h3>
            </div>
        </div>
    </div>
    </div>
</div>

<!-- ABOUT US -->
<div class="about">
    <div class="container">
        <div class="about-title">
            <h1>About Us</h1>
            <h2>Proin iaculis purus consequat sem cure</h2>
        </div>
        <div class="about-items">
            <div class="about-item">
                <div class="about-text about-text-right">
                    <h1>July 2010</h1>
                    <h2>Our Humble Beginnings</h2>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt. Proin 
                        iaculis purus consequat sem cure digni 
                        ssim. Donec porttitora entum suscipit.</p>
                </div>
                <img src="img/About_Us/item-1.jpg" alt="">
            </div>
            <div class="about-item about-item-right">
                <img src="img/About_Us/item-2.jpg" alt="">
                <div class="about-text about-text-left">
                    <h1>January 2011</h1>
                    <h2>Facing Startup Battles</h2>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt. Proin 
                        iaculis purus consequat sem cure digni 
                        ssim. Donec porttitora entum suscipit.</p>
                </div>
            </div>
            <div class="about-item">
                <div class="about-text about-text-right">
                    <h1>December 2012</h1>
                    <h2>Enter The Dark Days</h2>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt. Proin 
                        iaculis purus consequat sem cure digni 
                        ssim. Donec porttitora entum suscipit.</p>
                </div>
                <img src="img/About_Us/item-3.jpg" alt="">
            </div>
            <div class="about-item about-item-right">
                <img src="img/About_Us/item-4.jpg" alt="">
                <div class="about-text about-text-left">
                    <h1>february 2014</h1>
                    <h2>Our Triumph</h2>
                    <p>Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt. Proin 
                        iaculis purus consequat sem cure digni 
                        ssim. Donec porttitora entum suscipit.</p>
                </div>
            </div>
            <div class="about-item about-item-right">
                <img src="img/About_Us/item-5.jpg" alt="">
            </div>
        </div>
    </div>
</div>

<!-- TEAM -->
<div class="team">
    <div class="container">
        <div class="team-title">
            <h1>our amazing team</h1>
            <h2>Proin iaculis purus consequat sem cure</h2>
        </div>
        <div class="team-items">
            <div class="team-item">
                <img src="img/Team/worker.jpg" alt="">
                <div class="team-item-text">
                    <h1>Kimberly Thompson</h1>
                    <h2>Marketer</h2>
                </div>
                <div class="team-social">
                    <img src="img/Team/icons/twitter.jpg" alt="">
                    <img src="img/Team/icons/facebook.jpg" alt="">
                    <img src="img/Team/icons/pinterest.jpg" alt="">
                </div>
            </div>
            <div class="team-item">
                <img src="img/Team/worker.jpg" alt="">
                <div class="team-item-text">
                    <h1>Ricco Massimo</h1>
                    <h2>Coder</h2>
                </div>
                <div class="team-social">
                    <img src="img/Team/icons/twitter.jpg" alt="">
                    <img src="img/Team/icons/facebook.jpg" alt="">
                    <img src="img/Team/icons/google.jpg" alt="">
                </div>
            </div>
            <div class="team-item">
                <img src="img/Team/worker.jpg" alt="">
                <div class="team-item-text">
                    <h1>Uku Mason</h1>
                    <h2>Graphic Designer</h2>
                </div>
                <div class="team-social">
                    <img src="img/Team/icons/facebook.jpg" alt="">
                    <img src="img/Team/icons/pinterest.jpg" alt="">
                </div>
            </div> 
        </div>
        <div class="team-text">
            <p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
                aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
        </div>
    </div>
</div>

<!-- LOGOS -->
<div class="container">
    <div class="logos">
        <div class="logo">
            <img src="img/Logos/log-1.jpg" alt="">
        </div>
        <div class="logo">
            <img src="img/Logos/log-2.jpg" alt="">
        </div>
        <div class="logo">
            <img src="img/Logos/log-3.jpg" alt="">
        </div>
        <div class="logo">
            <img src="img/Logos/log-4.jpg" alt="">
        </div>
    </div>
</div>

<!-- CONTACT -->
<div class="contact">
    <div class="container">
        <div class="contact-title">
            <h1>Contact Us</h1>
            <h2>Proin iaculis purus consequat sem cure</h2>
        </div>
        <div class="forms">
        <div class="message-left">
            <input class="input" type="text" placeholder="YOUR NAME*">
            <input class="input" type="text" placeholder="YOUR E-MAIL*">
            <input class="input" type="text" placeholder="SUBJECT">
        </div>
        <div class="message-right">
            <input class="input-right" type="text" placeholder="YOUR MESSAGE*">
        </div>
        </div>
        <div class="btn-contact">
            <a href="#" class="btn">send message</a>
        </div>
    </div>
</div>

<!-- Footer -->
<div class="footer">
    <div class="container">
        <div class="footer-items">
        <div class="footer-text">Â© Copyright 2014 FreebiesXpress.com</div>
        <div class="social-icons">
            <img src="img/Footer/twitter-icon.jpg" alt="">
            <img src="img/Footer/facebook-icon.jpg" alt="">
            <img src="img/Footer/pinterest-icon.jpg" alt="">
            <img src="img/Footer/google-icon.jpg" alt="">
        </div>
    </div>
    </div>
</div>

</body>
</html>